runtime/etc/Makefile.mli-static

```diff
@@ -0,0 +1,81 @@
+# Copyright 2004-2019 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Makefile.mli-static
+#
+
+#
+# TODO
+#
+# There's a slightly more complicated build process going on here, because we
+# have 3 deliverables (for now):
+#
+#   - A client facing library file (static or shared)
+#   - A multi-locale launcher
+#   - A multi-locale executable
+#
+# We build the server as if it's a multi-locale executable, and then build the
+# client as if it's a (static, for this file) library.
```

Comment:
I don't see a corresponding Makefile.mli-shared.  Did you forget to add it, or is this a TODO that should be worked on as part of our larger effort?

Reply:
This would be a TODO, and along with that we'd need to make sure that this Makefile is strictly conforming to `static`.